neo4j/mapper/code_lists.go

```diff
@@ -3,29 +3,26 @@ package mapper
 import (
 	"strings"
 
-	"github.com/ONSdigital/dp-code-list-api/models"
 	"github.com/ONSdigital/dp-graph/graph/driver"
+	"github.com/ONSdigital/dp-graph/models"
 )
 
 //CodeLists returns a dpbolt.ResultMapper which converts a dpbolt.Result to models.CodeLists
 func CodeLists(codeLists *models.CodeListResults) ResultMapper {
 	return func(r *Result) error {
-		var label string
+		var id string
 		for _, v := range r.Data[0].([]interface{}) {
 			s := v.(string)
 			if strings.Contains(s, "_code_list_") {
-				label = strings.Replace(s, "_code_list_", "", -1)
+				id = strings.Replace(s, "_code_list_", "", -1)
```

Comment:
in a neo4j context this really is the label string that we're parsing. it can certainly be set as an ID later in the append, but i wonder if the variable name should be clearer - this is not a db id that we're doing a string replacement on. though tbf it probably wasnt clear enough that it was a db label beforehand either.

Reply:
This makes more sense to me:

```
var codeListID string
		for _, v := range r.Data[0].([]interface{}) {
			label := v.(string)
			if strings.Contains(label, "_code_list_") {
				// retrieve the codelist id from label
				codeListID = strings.Replace(label, "_code_list_", "", -1)
				break
			}
		}

		codeLists.Items = append(codeLists.Items, models.CodeList{
			ID: codeListID,
		})
```

What do you think?

Reply:
Please ignore the bad indentation above

Reply:
yes that would be very clear 👍